src/Kinepolis/AuthenticatedKinepolisClient.php

```diff
@@ -72,6 +73,33 @@ public function getMovieDetail(string $token, int $mid): array
         return $contents['movies'][0];
     }
 
+    public function getImage(string $token, string $path): UploadedFile
+    {
+        $lastSlashPosition = strrpos($path, '/');
+
+        $fileName = substr($path, $lastSlashPosition + 1);
+
+        $headers = $this->createHeaders($token);
+        $headers['Accept'] = 'image/jpeg';
+        $request = new Request(
+            'GET',
+            $this->movieApiBaseUrl . $path,
+            $headers
+        );
+
+        $response = $this->client->sendRequest($request);
+        $imageStream = $response->getBody();
+        $mimeType = $response->getHeaderLine('Content-Type');
+
+        return new UploadedFile(
```

Comment:
I'm a bit confused by the usage of the `UploadedFile` class here. Normally an `UploadedFile` is something that was uploaded to the UDB3 server and is physically present on the file system. In this case, it is a stream from another online URL, will this work?

Reply:
I tested it locally, and it worked, I will have to test it again, when it is on the server.

Reply:
Then it will be fine because it is a stream object.